docs/modules/documentation/containers/datetime-picker/examples/datetime-example/datetime-example.component.html

```diff
@@ -2,5 +2,5 @@
 <fd-datetime-picker [(ngModel)]="this.date"></fd-datetime-picker>
 
 <span style="padding-left: 20px;">
-    Selected: {{date.toLocaleString()}}
+    Selected: {{getSelectedDate()}}
```

Comment:
How about instead you do `Selected: {{date ? date.toLocaleString() : 'null'}}` ?

That allows removing the method in the ts file, simplifying the whole thing.